docs/adfs/README.md

```diff
@@ -57,6 +58,8 @@ passport.use(
       racComparison: "exact", // default to exact RequestedAuthnContext Comparison Type
       // From the metadata document
       audience: "https://adfs.acme_tools.com/FederationMetadata/2007-06/FederationMetadata.xml",
+      // otherwise it will thow because of invalid document signature
+      wantAuthnResponseSigned: false,
```

Comment:
We don't want any example code to disable security features.

Reply:
But it is very confusing  since this  is the exact configuration for AD FS and not working out of the box.
If possible it could also be mentioned how to adjust the ADFS configuration to work with it but for me there was no other oppertunity than setting `wantAuthnResponseSigned`to false